corehq/apps/hqwebapp/tests/test_default_landing_page.py

```diff
@@ -110,7 +110,7 @@ def test_web_user_landing_page(self, role, expected_urlname, enabled_toggle=None
     self.addCleanup(user.delete, deleted_by=None)
     self.client.login(username=user.username, password=self.global_password)
 
-    context = flag_enabled(enabled_toggle) if enabled_toggle else contextlib.nullcontext()
+    context = flag_enabled(enabled_toggle) if enabled_toggle else contextlib.suppress()
```

Comment:
I don't quite follow what this change is for - is there an exception you're expecting might get thrown by the `self.client.get`?

Reply:
no, this is just a no-op context manager. In Python >3.7 this would be `contextlib.nullcontext()`. Since there is no exception passed to the constructor it won't actually suppress anything even if there is an exception.

I've put in a comment to indicate that the intention